packages/ui-library/src/components/form-caption-group/index.ts

```diff
@@ -10,12 +10,12 @@ import { TAG_NAME } from './renderFunction';
 export class BlrFormCaptionGroup extends LitElement {
   static styles = [formCaptionGroupStyle];
 
-  @property() size: FormSizesType = 'md';
+  @property() sizeVariant: FormSizesType = 'md';
 
   protected render() {
     const classes = classMap({
       'blr-form-caption-group': true,
-      [this.size]: this.size,
+      [this.sizeVariant]: this.sizeVariant,
```

Comment:
Do we know if the form caption component also needs 'size' to be renamed to 'sizeVariant'?

Reply:
Yes, we do, but it seems that there is not issue for this. @angelicahoyss , can you create one prop alignment issue for the Form Caption as well?

Reply:
@thrbnhrtmnn @davidken91  Yes of course :) Here it is - https://github.com/deven-org/boiler/issues/1011